packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -30,12 +30,17 @@ const BlockInspector = ( {
 	selectedBlockClientId,
 	selectedBlockName,
 	showNoBlockSelectedMessage = true,
+	bubblesVirtually = true,
 } ) => {
 	if ( count > 1 ) {
 		return (
 			<div className="block-editor-block-inspector">
 				<MultiSelectionInspector />
-				<InspectorControls.Slot bubblesVirtually />
+				{ bubblesVirtually ? (
+					<InspectorControls.Slot bubblesVirtually />
+				) : (
+					<InspectorControls.Slot />
+				) }
```

Comment:
Could these be written without the ternary?
```suggestion
				<InspectorControls.Slot bubblesVirtually={ bubblesVirtually } />
```

Reply:
Oh, yes, of course!